SOPT-iOS/Projects/Features/StampFeature/Sources/ListDetailScene/ViewModel/ListDetailViewModel.swift

```diff
@@ -154,19 +158,22 @@ extension ListDetailViewModel {
     
     listDetailModel.asDriver()
       .compactMap {
+        self.uploadedUrl = $0.image
```

Comment:
release브랜치에 반영되었던 부분인데 혹시 release 브랜치에서 새 브랜치 파신게 맞는지 확인 한 번 더 부탁드립니다!_!

Reply:
그리고 이 부분에서도 self로 뷰모델을 참조하는데 다시 self의 cancelBag에 저장되기 때문에 순환참조가 발생할 것 같습니다 ! 
```suggestion
    listDetailModel.asDrivaer()
        .withUnretained(self)
        .compactMap { owner, model in
            self.uploadedUrl = model.image
            self.stampId = model.stampId
            return model
        }
```

Reply:
> release브랜치에 반영되었던 부분인데 혹시 release 브랜치에서 새 브랜치 파신게 맞는지 확인 한 번 더 부탁드립니다!_!
🥹 실수입니다 .. 잘 되돌려놓겠습니다..